src/util/Formatters.js

```diff
@@ -13,13 +15,134 @@ const {
   underscore,
 } = require('@discordjs/builders');
 
-exports.blockQuote = blockQuote;
-exports.bold = bold;
-exports.codeBlock = codeBlock;
-exports.inlineCode = inlineCode;
-exports.italic = italic;
-exports.quote = quote;
-exports.strikethrough = strikethrough;
-exports.time = time;
-exports.TimestampStyles = TimestampStyles;
-exports.underscore = underscore;
+/**
+ * Contains various Discord-specific functions for formatting messages.
+ */
+
```

Comment:
We don't seem to have an empty line between the class and its jsdoc anywhere else
```suggestion
```

Reply:
That's an oversight when I was trying multiple things (see previous commits), thanks for catching this one 😅